src/test/java/uk/gov/hmcts/reform/userprofileapi/util/IdamStatusResolverTest.java

```diff
@@ -34,32 +41,35 @@
 
     @Test
     public void should_return_error_message_by_HttpStatus_provided() {
-        String httpStatusString = IdamStatusResolver.resolveStatusAndReturnMessage(HttpStatus.OK);
+        String httpStatusString = resolveStatusAndReturnMessage(HttpStatus.OK);
         assertThat(httpStatusString).isEqualTo(OK);
 
-        httpStatusString = IdamStatusResolver.resolveStatusAndReturnMessage(HttpStatus.CREATED);
+        httpStatusString = resolveStatusAndReturnMessage(HttpStatus.CREATED);
         assertThat(httpStatusString).isEqualTo(ACCEPTED);
 
-        httpStatusString = IdamStatusResolver.resolveStatusAndReturnMessage(HttpStatus.BAD_REQUEST);
+        httpStatusString = resolveStatusAndReturnMessage(HttpStatus.BAD_REQUEST);
         assertThat(httpStatusString).isEqualTo(INVALID_REQUEST);
 
-        httpStatusString = IdamStatusResolver.resolveStatusAndReturnMessage(HttpStatus.UNAUTHORIZED);
+        httpStatusString = resolveStatusAndReturnMessage(HttpStatus.UNAUTHORIZED);
         assertThat(httpStatusString).isEqualTo(MISSING_TOKEN);
```

Comment:
prefix all the test classes with test in this class

Reply:
Its not mandatory that test case name should start with Test or is it Hmcts mandate ?

Reply:
before adding comments please you go through the email QA assurance: John review comments and let comeback on this.

Reply:
please forward the email to team if you have ,since I don't have , better for team to refer what naming conventions he has mentioned.  We can have tech debt for changing method name of all tcs in all 3 repos then.

Reply:
search your email box  you will get and we are already following pattern every where like test_

Reply:
Created RDCC-1703 for tech debt , where all team will discuss the finalised naming convention of test cases and will be implemented for all 3 repos.